OpenNefia.Core/Locale/LocalizationManager.cs

```diff
@@ -216,5 +223,47 @@ public bool TryGetLocalizationData(EntityUid uid, [NotNullWhen(true)] out LuaTab
 
             return TryGetTable($"OpenNefia.Prototypes.Entity.{metadata.EntityPrototype.ID}", out table);
         }
+
+        private void MungeEcksdee()
+        {
+            _entityCache.Clear();
+            return;
+            if (!TryGetTable("OpenNefia.Prototypes.Entity.Elona", out LuaTable? entityTable))
+            {
+                return;
+            }
+
+            foreach(KeyValuePair<object, object> pair in entityTable)
+            {
+                if (pair.Value is not LuaTable table)
+                    continue;
+                if (!table.TryGetTable("_LocData", out var locTable))
```

Comment:
Make a `private const string` for `"_LocData"`.

Reply:
added in 0845470